### PR TITLE
Improve the pagination help

### PR DESCRIPTION
**Pull request by joshbean**
This is my attempt to clarify the help for the AWS CLI's standardized pagination options.  The way to use these options is well documented in the AWS CLI User Guide, but the inline help didn't help me understand the difference between max-items and page-size.

### CODE REVIEW

File: awscli/customizations/paginate.py [hunk 1]
```diff
@@ -39,19 +39,37 @@
 STARTING_TOKEN_HELP = """
 <p>A token to specify where to start paginating.  This is the
 <code>NextToken</code> from a previously truncated response.</p>
+<p>For more information, see <a
```
Comment (JordonPhillips) [inline awscli/customizations/paginate.py]: I'm not sure I want to link to this page as it is a bit out of date. If it were updated I may be inclined, but I would prefer that it read "for detailed examples".

Comment (joshbean) [inline awscli/customizations/paginate.py]: I can make that change. How about "For usage examples, see ...". Does that work?

Comment (JordonPhillips) [inline awscli/customizations/paginate.py]: Sure, that works.

Comment (joshbean) [inline awscli/customizations/paginate.py]: Done. I just pushed a new commit with that change. Thanks for the fast review!

**Comment by joshbean** [comment]
Bump. I'd love to get a second set of eyes on this. It would be really great to get this clarification into the official documentation.

**Pull request merged**